sdk/python/symbolchain/BasicTransactionFactory.py

```diff
@@ -0,0 +1,29 @@
+from .CryptoTypes import PublicKey
+from .TransactionDescriptorProcessor import TransactionDescriptorProcessor
+from .TypeParserBuilder import build_type_hints_map
+
+
+class BasicTransactionFactory:
```

Comment:
@Jaguar0625  does it make sense having any tests for this one instead of both transaction factories?